index.bs

```diff
@@ -6871,7 +6872,7 @@ The <code>network.Cookie</code> type represents a cookie.
 To <dfn>serialize cookie</dfn> given |stored cookie|:
 
 Note: The definitions of |stored cookie|'s fields are from [[COOKIES]], except
-samesite-flag, which is from [[SAME-SITE-COOKIES]].
+same-site-flag, which is from [[SAME-SITE-COOKIES]].
```

Comment:
This is wrong now, I think; we're just following RFC6265bis everywhere?

Reply:
Ah, alright, I removed it now. I was just not sure how all these overrides work